drs_downloader/cli.py

```diff
@@ -1,11 +1,124 @@
+import logging
+import uuid
+from pathlib import Path
+from typing import List
+
 import click
+import pandas as pd
+
+from drs_downloader.clients.mock import MockDrsClient
+from drs_downloader.clients.terra import TerraDrsClient
+from drs_downloader.manager import DrsAsyncManager
+
+logging.basicConfig(format='%(asctime)s %(message)s',  encoding='utf-8', level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# these control our simulation
+NUMBER_OF_OBJECT_IDS = 10
 
 
-@click.command()
+@click.group()
 def cli():
-    """Welcome to drs_downloader."""
-    print(cli.__doc__)
+    """Copy DRS objects from the cloud to your local system ."""
+    pass
+
+
+@cli.command()
+@click.option("--silent", "-s", is_flag=True, show_default=True, default=False, help="Display nothing.")
+@click.option("--destination_dir", "-d", show_default=True, default='/tmp/testing', help="Destination directory.")
+def mock(silent: bool, destination_dir: str):
```

Comment:
I'm assuming this function is purely used for demonstration/testing purposes  Since there is no actual signing/downloading going on right now.

Reply:
Please see https://github.com/anvilproject/drs_downloader/blob/feature/manager/drs_downloader/clients/terra.py